behat.yml

```diff
@@ -283,4 +283,9 @@ default:
 
     meta_service_dependency_configuration:
       paths: [ %paths.base%/features/MetaServiceDependencyConfiguration.feature ]
-      contexts: [ MetaServiceDependencyConfigurationContext ]
\ No newline at end of file
+      contexts: [ MetaServiceDependencyConfigurationContext ]
+    
+    test_proxy_configuration:
+      paths: [ %paths.base%/features/TestProxyConfiguration.feature ]
+      contexts: [ TestProxyConfigurationContext ]
+      
```

Comment:
Useless whitespaces.

Reply:
you did not ask for a Missing \n in the last comment?